syntax = "proto3";

package token;

enum ServerType {
  MASTER = 0;
  WORKER = 1;
}

message ServerIdentity {
  ServerType server_type = 1;
  uint64 server_index = 2;
}

message PingMessage { ServerIdentity server_identity = 1; }